### ISSUE TITLE
Scientific linux VScode install fail glibc version and libXss.so.1

### ISSUE THREAD

**Issue by anlinx86** [Microsoft/vscode#14656]
- VSCode Version: code-1.6.1
- OS Version: 2.6.32-642.6.2.el6.x86_64  / Red Hat 4.4.7-17

System
` $ gcc --version`
> gcc (GCC) 4.9.1 20140922 (Red Hat 4.9.1-10)

achieved with Red Hat Software Collections / devtoolset-3 which ships with:

`$ ldd --version`
ldd (GNU libc) **2.12**

- Install via rpm:

`$ rpm -qpR code-1.6.1-1476373175.el7.x86_64.rpm`
glibc >= 2.15
libXss.so.1
/bin/sh
/bin/sh
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(CompressedFileNames) <= 3.0.4-1

`$ rpm -Uvh code-1.6.1-1476373175.el7.x86_64.rpm`
error: Failed dependencies:
glibc >= 2.15 is needed by code-1.6.1-1476373175.el7.x86_64
libXss.so.1 is needed by code-1.6.1-1476373175.el7.x86_64

- Install via yum:
 
`$ yum install code-1.6.1-1476373175.el7.x86_64.rpm`

Loaded plugins: refresh-packagekit, security
Setting up Install Process
Examining code-1.6.1-1476373175.el7.x86_64.rpm: code-1.6.1-1476373175.el7.x86_64
Marking code-1.6.1-1476373175.el7.x86_64.rpm to be installed
Resolving Dependencies
--> Running transaction check
---> Package code.x86_64 0:1.6.1-1476373175.el7 will be installed
--> Processing Dependency: glibc >= 2.15 for package: code-1.6.1-1476373175.el7.x86_64
--> Processing Dependency: libXss.so.1 for package: code-1.6.1-1476373175.el7.x86_64
--> Running transaction check
---> Package code.x86_64 0:1.6.1-1476373175.el7 will be installed
--> Processing Dependency: glibc >= 2.15 for package: code-1.6.1-1476373175.el7.x86_64
---> Package libXScrnSaver.i686 0:1.2.2-2.el6 will be installed
--> Finished Dependency Resolution
Error: Package: code-1.6.1-1476373175.el7.x86_64 (/code-1.6.1-1476373175.el7.x86_64)
           Requires: glibc >= 2.15
           Installed: glibc-2.12-1.192.el6.i686 (@sl)
               glibc = 2.12-1.192.el6
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest `

Are there any possible fixes that avoid the risky road of upgrading `glibc`?

**Comment by josteink** [comment]
I was  getting the same on Fedora 24:

```
$ gcc --version                                   
gcc (GCC) 6.2.1 20160916 (Red Hat 6.2.1-2)
Copyright (C) 2016 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

$ ldd --version                               
ldd (GNU libc) 2.23
Copyright (C) 2016 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
$ sudo rpm -U code-1.6.1-1476373175.el7.x86_64.rpm
error: Failed dependencies:
	libXss.so.1 is needed by code-1.6.1-1476373175.el7.x86_64
````

However your trick about using `yum` to install the RPM solved it for me.

````
$ sudo yum install code-1.6.1-1476373175.el7.x86_64.rpm   
Dependencies resolved.
=================================================================================================
 Package               Arch           Version                         Repository            Size
=================================================================================================
Installing:
 libXScrnSaver         i686           1.2.2-10.fc24                   fedora                28 k
Upgrading:
 code                  x86_64         1.6.1-1476373175.el7            @commandline          48 M

Transaction Summary
=================================================================================================
Install  1 Package
Upgrade  1 Package

Total size: 48 M
Total download size: 28 k
Is this ok [y/N]: y
````

So if nothing else comes out of it this, at least know that your issue gave me as a Fedora user something else to try, which actually resolved a very similar case :)

**Comment by Tyriar** [comment]
You must install via `yum` to also install the dependencies, and [we require glibc 2.15+](https://code.visualstudio.com/Docs/supporting/requirements). There's a bit of history to this in #1056, #3603 and #9487. I'm not sure if it's impossible to get running on the old version but we can't justify the resources required for further investigation.

I'll close this as a duplicate of #3603
